Source/Renderer/AutomaticUniforms.js

```diff
@@ -1409,6 +1409,30 @@ define([
         }),
 
         /**
+         * An automatic GLSL uniform representing the current scene background color.
+         *
+         * @alias czm_backgroundColor
+         * @glslUniform
+         *
+         * @example
+         * // GLSL declaration
+         * uniform vec4 czm_backgroundColor;
+         *
+         * // Example: In fragment shader, if frag color matches background, invert frag color.
+         * if (czm_backgroundColor.rgb == gl_FragColor.rgb)
```

Comment:
I know this is a code example, but are you sure you are allowed to read from `gl_FragColor` (without prior writing to it).  I do not think this is defined in the GLSL spec.